docs/local_resource.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Run Local and/or Occasional Workflows with Local Resource
+layout: docs
+---
+A Local Resource works like any other resource in your sidebar (which are most
+often some combination of Docker Build instructions and K8s Yaml)--encompasses
```

Comment:
nit: "K8s Yaml" capitalization feels weird, esp. given "k8s yaml" a few lines down. I think it should be "YAML" or "yaml". I don't have an opinion and don't know if we have a docs standard on k8s vs K8s (it looks like we currently have 5 of the former and 2 of the latter), but it'd be good to be consistent within this doc

Reply:
you are 100% correct ty